docs/frontends/webapi.rst

```diff
@@ -1725,6 +1725,9 @@ incorrectly.
   storage-index: list of (base32) storage index strings
   stats: a dictionary with the same keys as the t=start-deep-stats command
          (described below)
+  api-version (int): deep-manifest API version. Will be increased every
+               time backwards-incompatible change is introduced. Current version
+               is 1.
```

Comment:
I wonder if we can provide any tools to make it more likely this statement (middle sentence) is actually true.

Reply:
Something like asserting against the keys in the resulting mapping so that there is a failing test if that set changes in a way that merits a version bump (on the premise that a failing test with the right comment/docstring will prompt a developer to bump the version number)?  This doesn't exactly prove backwards incompatible changes must come with a version bump but it might better than nothing.

Reply:
The claim is also that you can add keys if you want, you just can't change the type/meaning of an existing key -- that is, "add a new key" is NOT backwards-incompatible. Soooo...hmm, I guess I'd be a little worried an automated thing would have us increment the version on any change at all? But, perhaps some words like the above in said unit-test would make it easier to decide what to do (i.e. change the unit-test or increment version)